simulation/halsim_gui/src/main/native/cpp/DriverStationGui.cpp

```diff
@@ -675,6 +683,8 @@ static void DriverStationOptionMenu() {
     ImGui::MenuItem("Turn off DS (real DS connected)", nullptr, true, false);
   } else {
     ImGui::MenuItem("Turn off DS", nullptr, &gDisableDS);
+    ImGui::MenuItem("Turn off missing joysticks", nullptr,
```

Comment:
I'm worried this description is confusing.  Maybe make it default to true and have it be "Zero disconnected joysticks"?

Reply:
> I'm worried this description is confusing. Maybe make it default to true and have it be "Zero disconnected joysticks"?

Done.